index.js

```diff
@@ -289,7 +289,7 @@ const createClient = (profile, request = _request) => {
 		})
 	}
 
-	const radar = (north, west, south, east, opt) => {
+	const radar = ({north, west, south, east, opt}) => {
```

Comment:
This should be `({north, west, south, east}, opt) => {}` instead, right?

Reply:
yup. will fix it.